<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="onePlayerVsAIButton">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>410</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>1 Player</string>
   </property>
  </widget>
  <widget class="QPushButton" name="twoPlayersButton">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>450</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>2 Players</string>
   </property>
  </widget>
  <widget class="QPushButton" name="profileButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>200</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="infoButton">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>90</y>
     <width>200</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>info</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
